pkg/mqtt-paho/contrib/riot_iface.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2019  Javier FILEIV <javier.fileiv@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @brief       MQTT common RIOT interface functions
+ *
+ * @author      Javier FILEIV <javier.fileiv@gmail.com>
+ */
+#include <string.h>
+#include <net/ipv6/addr.h>
+#include <net/af.h>
+#include <net/sock.h>
+#include <net/sock/tcp.h>
+#include <mqtt.h>
+#include <log.h>
+
+#ifndef MQTT_THREAD_PRIORITY
+#define MQTT_THREAD_PRIORITY    (THREAD_PRIORITY_MAIN - 4)
+#endif
+
+#ifndef MQTT_THREAD_STACKSIZE
+#define MQTT_THREAD_STACKSIZE   (THREAD_STACKSIZE_DEFAULT)
```

Comment:
This define is unused. I guess it is supposed to be used [here](https://github.com/RIOT-OS/RIOT/pull/12647/files#diff-c22ca7429a5f05a0b066f97aebc963b7R154).